lib/Doctrine/DBAL/Query/QueryBuilder.php

```diff
@@ -1088,8 +1156,7 @@ private function getFromClauses() : array
         $knownAliases = [];
 
         // Loop through all FROM clauses
```

Comment:
Looks useless.

Reply:
What do you suggest?

Reply:
Please remove it.

Reply:
Oh, you mean the comment. Done in d3ba7e67c3e9c27b6efef48ed507b36c735a1a7d.